src/sentry/snuba/metrics/extraction.py

```diff
@@ -565,22 +565,22 @@ class SupportedBy:
     on_demand_metrics: bool
 
     @classmethod
-    def neither(cls):
+    def neither(cls) -> SupportedBy:
```

Comment:
`-> Self` is slightly more accurate (imported from `typing`)

Reply:
Ah nice. Didn't know it was a thing.